### ISSUE TITLE
EDSC-981: Blank granule filter params results in 500 errors for multiple collections in a project

### ISSUE THREAD

**Issue by earthdata-tasktop** [nasa/edsc-search-test-config-3#339]
Steps to re-produce (in OPS):

1. Find and add two collections that have granules to a project
   * [https://search.earthdata.nasa.gov/search/project?p=!C14758250-LPDAAC_ECS!C119124186-NSIDC_ECS](https://search.earthdata.nasa.gov/search/project?p=!C14758250-LPDAAC_ECS!C119124186-NSIDC_ECS)
2. Open the granule filter of the ***second*** collection and set a temporal.
3. Click Apply to close the filter panel
   * The granule filter icon doesn't change to a state which indicates some filter constraints are set.
4. Click "Retrieve Project Data"
   * An empty (white) page will be shown

**Comment by earthdata-tasktop** [comment]
(Comment from Bo Sun):

For example, set temporal 2011-03-03T00:00:00Z and 2011-03-03T23:59:59Z for collection AE_Rain, &pg[]= is inserted to the url which causes the error:  

[https://search.earthdata.nasa.gov/search/project?p=!C14758250-LPDAAC_ECS!C119124186-NSIDC_ECS&amp;amp;pg](https://search.earthdata.nasa.gov/search/project?p=!C14758250-LPDAAC_ECS!C119124186-NSIDC_ECS&pg)[]=&amp;amp;pg[2][qt]=2011-03-03T00%3A00%3A00.000Z%2C2011-03-03T23%3A59%3A59.000Z

**Comment by earthdata-tasktop** [comment]
(Comment from Bo Sun):

##### Steps to Verify

1. Find two collections that have granuels.
2. Add both collections to a project
3. View Project
4. Open the granule filter of the ***second*** granule
5. Change one of the filters, i.e. Day/Night flag.
6. You shouldn't see invalid query param &pg[]=

**Comment by earthdata-tasktop** [comment]
(Comment from Kathleen Carr):

I used OPS to reproduce the problem.

On my Mac, using Safari, I got an empty white page.

On my PC, using IE 10,  

I got a 400 error.  In addition, the granule filter icon did not change to indicate that filter constraints were applied. (see attached screen shots from OPS)

**Comment by earthdata-tasktop** [comment]
(Comment from Kathleen Carr):

In UAT, I found two MODIS collections with granules and added them to a project.  

I then clicked on View Project, and then clicked on the 'Show Granule Filters' icon for the second collection in the project.   I set a filter (I tried both temporal and Day/Night filters) and clicked Apply.

The number of granules shown for the collection decreased, and there was a blue outline around the 'Show Granule Filters' icon.  (see attached screen shot)

I then clicked on 'Retrieve Project Data', and I saw the Data Access screen.   On the Data Access screen, the number of granules shown for the second collection was the number of granules after the filter was applied.
